src/components/Fruit.js

```diff
@@ -0,0 +1,40 @@
+import './Fruit.css';
+
+function Fruit({image, title, fruitAmount, fruitSetter}) {
+    const increase = () => {
+        fruitSetter(fruitAmount => fruitAmount + 1)
+    }
+
+    const decrease = () => {
+        fruitAmount > 0? fruitSetter(fruitAmount => fruitAmount -1): fruitSetter(0)
```

Comment:
Goed dat je gebruikmaakt van de huidige waarde van de set functie. Echter is de variabelnaam al eerder gebruikt, dus moet je deze wel hernoemen